frontend-demo/src/services/ai/user/queries.ts

```diff
@@ -17,7 +17,7 @@ const queries = {
       queryKey: keys.me(),
       queryFn: (): Promise<UserResponse> => {
         return axiosInstance
-          .get(`${AI_API_BASE_URL}/me`)
+          .get(`${AI_API_BASE_URL}/me/`)
```

Comment:
In RC, this is currently hitting https://api.rc.learn.mit.edu/ai/me

Due to some apisix shenanigans, this is being re-routed to https://api.rc.learn.mit.edu/me/.

Adding the trailing slash prevents this re-route.

Related: https://github.com/mitodl/hq/issues/7139